CHANGELOG.rst

```diff
@@ -1,6 +1,13 @@
 Changelog
 =========
 
+3.1.0 (2025-04-23)
+-------------------
+* Updated the PHS model in compliance with the ISO 7933:2023 standard
+    - Added default‑kwarg overrides for 2023 mode (f_r, t_re, t_cr_eq)
+    - removed unused variable `round` from `default_kwargs`
+* Included test cases according to the ISO 7933:2023 standard
```

Comment:
please add a bullet point saying that test case 3 from the standard is not passing.